README.md

```diff
@@ -0,0 +1,136 @@
+### Notebook Images
+
+These images were created to be used with Open Data Hub (ODH) or Red Hat Openshift Data Science (RHODS) with the Kubeflow Notebook Controller as the launcher.
+
+The notebooks support CPU and GPU running modes. GPU notebooks can be also spawned in clusters without GPU enabled using the CPU capabilities:
+
+**CPU Notebooks**
+- Minimal
+- Standard Data Science
+
+**GPU Notebooks**
+- CUDA
+- CUDA PyTorch
+- CUDA TensorFlow
+
+
+#### Folder Structure of the Repository
+
+```
+.
+├── base
+│   └── ubi8-python-3.8
+│       └── Dockerfile		#Includes the instructions for the base image
+├── LICENSE
+├── Makefile                #Lists shell commands for the build and push process
+├── OWNERS                  #Includes the mainteiners of the repository
+└── README.md               #Includes basic information about the repository
+
+```
+
+#### Container Image Layering
+In order to avoid packages conflicts and improve the performance of the building process, the notebooks at the bottom are built using the previous notebook as their base image:
+
+```mermaid
+flowchart TD
+    ubi/python-3.8 --> python-3.8-minimal;
+    python-3.8-minimal --> python-3.8-standar-data-science;
+    python-3.8-minimal --> python-3.8-cuda;
+    python-3.8-cuda --> python-3.8-cuda-pytorch;
+    python-3.8-cuda --> python-3.8-cuda-tensorflow;
+```    
```

Comment:
We only need a diagram that includes UBI + Base. Use https://github.com/guimou/custom-notebooks/tree/main/notebook-controller-images#ubi8-python-38-tree as a reference.